### PR TITLE
Updated menu. Removed deprecated telemetry. Renamed Delfin to monitor

### LINKED ISSUE #434: Consistent naming convention of Menu items to reduce confusion and remove obsolete menu items.
> > > How about just "Monitor" as of now?
> > 
> > 
> > We already have a Monitor menu item. It is used to link to the Grafana Dashboard and Alert Manager when Telemetry isinstalled. You can also configure the Prometheus, Grafana and Alert manager using the configuration page.
> 
> Can we simply call it “Storage Manager”?

> I think, resrouce, storage, all these confusing if you have multiple menu items.
> Can we not obsolete the exciting telemetry feature? Fully focus to provide the monitoring through delfin?

> @anvithks please raise an issue for this. It is not related to this PR anyways. Add the issue here. Then merge the PR

The current menu has items that are confusing, `Resources`, `Resource manager`, `Monitor` and `Infrastructure`.
To reduce this confusion and to make the function of each menu item clear we need to rename and remove some of the menu items.

The current suggestion is to remove Monitor (Telemetry feature), rename the existing Resource Manager to Monitor and move all monitoring to Delfin, the SODA SRM.

_Originally posted by @skdwriting in https://github.com/sodafoundation/dashboard/pull/426#issuecomment-699312375_

### PR DESCRIPTION
**Pull request by anvithks**
**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:
This PR removes access to deprecated Telemetry feature. It also renames the delfin menu items and route.

**Which issue(s) this PR fixes**:
Fixes #434 

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

image

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by anvithks** [comment]
@kumarashit @skdwriting 
updated the PR. 
Changed the path from `/monitor` to `/resource-monitor`.
image

**Review by kumarashit — APPROVED** [review]
LGTM

**Review by skdwriting — APPROVED** [review]
LGTM

File: entrypoint.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ OPENSDS_AUTH_URL=${OPENSDS_AUTH_URL:-http://127.0.0.1/identity}
 
 OPENSDS_HOTPOT_API_VERSION=${OPENSDS_HOTPOT_API_VERSION:-v1beta}
 OPENSDS_ORCHESTRATION_API_VERSION=${OPENSDS_ORCHESTRATION_API_VERSION:-orch}
-SODA_DELFIN_API_LOCATION=${SODA_DELFIN_API_LOCATION:-delfin}
+SODA_DELFIN_API_LOCATION=${SODA_DELFIN_API_LOCATION:-monitor}
```
[the remark this answers is not in the document]
Comment (anvithks) [reply entrypoint.sh]: It is changed. I think you are seeing the older commit. 
In the latest commit it is resource-monitor

**Pull request merged**